#
# Submitted-By: Erwin Rol, 2006-03-24
# Committed-By: Erwin Rol
#
# Error:
# The initial message asks for donations and this doesn't
# look very profesional on a commercial end product.
#
# Description:
#
# State:
# Unknown
#
--- initng-0.6.0RC2/plugins/cpout/initng_colorprint_out.c.orig	2006-03-23 15:26:44.000000000 +0100
+++ initng-0.6.0RC2/plugins/cpout/initng_colorprint_out.c	2006-03-23 15:27:20.000000000 +0100
@@ -405,11 +405,8 @@
     }
 
 
-    printf("\n" C_BLUE "\tNext Generation Init version ( %s )" C_OFF,
+    printf("\n" C_BLUE "\tNext Generation Init version ( %s )\n\n" C_OFF,
            INITNG_VERSION);
-    printf("\n" C_GREEN "\thttp://initng.thinktux.net\n" C_OFF);
-    printf("\tAuthor: Jimmy Wennlund <jimmy.wennlund@gmail.com>\n");
-    printf("\tIf you find initng useful, please consider a small donation.\n\n");
     D_("module_init();\n");
     lastservice = NULL;
     initng_plugin_hook_add(&g.ERR_MSG, 10, &cp_print_error);
